client/blocks/comments/form.scss

```diff
@@ -19,7 +19,7 @@
 	button:not(.components-button) {
 		opacity: 0;
 		position: absolute;
-		margin: 16px 0 -19px;
+		margin: 16px 0 0;
 		transition: all 0.2s ease-in-out;
```

Comment:
Since this margin issue is strictly on the post cards inline comments section, should we handle the style more specifically there?

Il try to test all instances i can find of the comments form, as right now it looks like we are updating the margin everywhere its used.

Reply:
Yeah, I looked around for instances where it might break things and couldn't find anything either. I'm not sure why this negative margin was ever applied. Guessing it was a legacy hack that no longer really applies.